syncano/models/options.py

```diff
@@ -95,7 +96,7 @@ def add_field(self, field):
             raise SyncanoValueError('Field "{0}" already defined'.format(field.name))
 
         self.field_names.append(field.name)
-        self.fields.append(field)
+        self.fields.insert(bisect(self.fields, field), field)
```

Comment:
Um.. bisect? Yeah, well, that would work. But why is it needed anyway? Why do you care about the order at all?